subprojects/core/src/integTest/groovy/org/gradle/api/services/BuildServiceIntegrationTest.groovy

```diff
@@ -250,29 +253,6 @@ service: closed with value 11
         outputDoesNotContain "'Task#usesService'"
     }
 
-    def "does not nag when service is annotated with @ServiceReference (unnamed) and feature preview is enabled"() {
```

Comment:
Covered by this data-driven test case: https://github.com/gradle/gradle/pull/22987/commits/2c7ff41169666513fced9e647cd1a7ea48fda2b1#diff-177b01df4f530a32fd73f07c48f1767fb6ce34465477be561e6a71f163434f36R339